src/DataTypes/PaymentMethod.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace CommerceGuys\AuthNet\DataTypes;
+
+interface PaymentMethod
```

Comment:
I'm typically used to Interface's being suffixed with Interface, can we rename this to `PaymentMethodInterface`? I thought that was a PSR-2 thing and should have been caught by phpcs.